src/renderer/screens/config/ConfigScreens/WorkspaceScreen.js

```diff
@@ -120,9 +122,43 @@ class WorkspaceScreen extends Component {
           <h4>TRUFFLE PROJECTS</h4>
           <div className="Row">
             <div className="RowItem">
+              {this.props.config.validationErrors["workspace.project"] && (
+                <div>
+                  <div className="ValidationError">
+                    {
+                      this.props.config.validationErrors["workspace.project"]
+                        .message
+                    }
+                    <button
+                      onClick={() => {
+                        this.setState({
```

Comment:
I wonder if this could be a class method instead, to preserve readability:

```js
toggleErrorDetails = () => this.setState({ showErrorDetails: !this.state.showErrorDetails })
```

And then your button can just be:

```js
<button onClick={this.toggleErrorDetails}>
```